homeassistant/components/rako/config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Config flow for Rako."""
+import asyncio
+from typing import Any, Dict, Optional
+
+from python_rako import Bridge, discover_bridge
+from python_rako.const import RAKO_BRIDGE_DEFAULT_PORT
+from python_rako.exceptions import RakoBridgeError
+from python_rako.model import BridgeInfo
+import voluptuous as vol
+
+from homeassistant.config_entries import CONN_CLASS_LOCAL_PUSH, ConfigFlow
+from homeassistant.const import CONF_BASE, CONF_HOST, CONF_PORT
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
Fixed in pylint 2.7.3.
```suggestion
from .const import DOMAIN
```